src/utils.jl

```diff
@@ -138,14 +141,18 @@ end
 """
     parsevalsum(uh, grid)
 
-Return `real(Σ uh)` on the `grid`, i.e.,
+Return the real part of the sum of `uh` on the `grid`. For example on a 2D grid,
+`parsevalsum` returns
 
 ```math
-ℜ [ \\sum_{𝐤} û_{𝐤} L_x L_y ] \\,,
+ℜ [ \\sum_{𝐤} û_{𝐤} L_x L_y ] ,
 
 ```
 where ``û_{𝐤} =`` `uh` `` / (n_x e^{i 𝐤 ⋅ 𝐱₀})``. The elements of the vector ``𝐱₀`` are the
 left-most position in each direction, e.g., for a 2D grid `(grid.x[1], grid.y[1])`.
+
+When the input `uh` comes from a real-FFT transform, then `parsevalsum` takes care to
+count twice the contribution from certain ``k``-modes.
```

Comment:
count the contribution from certain ``k``-modes twice.

Not necessary but maybe sounds better?